<launch>
    
    <node pkg="pointcloud_to_laserscan" name="pointcloud_to_laserscan_node"
    type="pointcloud_to_laserscan_node" output="screen">
        <param name="use_inf" value="false"/>
        <param name="concurrency_level" value="0"/>
        <param name="range_min" value="0"/>
    </node>

</launch>
